src/select.jsx

```diff
@@ -344,7 +344,7 @@ var classBase = React.createClass({
           </div>
           : ''
         }
-        <select name={this.props.selectName} value={this.state.selectedOptionVal} className={this.props.hiddenSelectClassName} tabIndex={-1} aria-hidden={true} >
+        <select name={this.props.selectName} onChange={() => {}} value={this.state.selectedOptionVal} className={this.props.hiddenSelectClassName} tabIndex={-1} aria-hidden={true} >
```

Comment:
hmm, I'm not sure how we want to handle this. I think we probably want `readOnly` since all of the interaction occurs with the divs and not the select element itself.  @alexlande what's the best way forward for accessibility?